config/config.py.sandbox

```diff
@@ -4,63 +4,65 @@ use_gcp_buckets = False
 red_bucket = ''
 green_bucket = ''
 
-sqlite_db = '/finngen/library-green/finngen_R9/finngen_R9_analysis_data/annotations/fgq.r9.1.db'
-basic_info_file = '/finngen/library-red/finngen_R9/phenotype_1.0/data/finngen_R9_gt_samples_info.txt.gz'
-intensity_files_location = '/finngen/library-red/finngen_R9/cluster_plot_1.0/data/'
+sqlite_db = '/finngen/library-green/finngen_R11/finngen_R11_analysis_data/annotations/fgq.r11.1.db'
```

Comment:
This file is fgq.R11.1.db not lowercase fgq.r11.1.db

Reply:
indeed! Fixed